backend/app/test/service/test_user_service.py

```diff
@@ -0,0 +1,148 @@
+        db.session.add(user_2)
+        db.session.commit()
+
+        mock.return_value = User(fb_id='123', email=None, name='Albert')
+        user_list, status_code = user_service.get_all_users()
+
+        expected = [{'email': None, 'name': 'Jane', 'id': 2, 'relationship_status': None}, {'email': None, 'name': 'Jake', 'id': 1, 'relationship_status': None}] 
+        
+        assert user_list == expected
+        assert status_code == 201
+
+    @cleared_db
+    @patch('app.main.service.user_service.get_current_user')
+    def test_create_user_follow_request(self, mock):
+        user_1 =  User(fb_id='123', email=None, name='Jake')
+        user_2 = User(fb_id='123', email=None, name='Jane')
+        
+        db.session.add(user_1)
+        db.session.add(user_2)
+        db.session.commit()
+
+        mock.return_value = user_1
+        resp, status_code = user_service.create_user_follow_request({"id": 2})
+
+        assert status_code == 201
+
+    @cleared_db
+    @patch('app.main.service.user_service.get_current_user')
+    def test_confirm_user_follow_request(self, mock):
+        user_1 =  User(fb_id='123', email=None, name='Jake')
+        user_2 = User(fb_id='123', email=None, name='Jane')
+        
+        user_1.followers.append(user_2)
+        db.session.add(user_1)
+        db.session.add(user_2)
+        db.session.commit()
+
+        mock.return_value = user_2
+
+        resp, status_code = user_service.confirm_user_follow_request({"id": 1})
+
+        assert status_code == 201
+
+    @cleared_db
+    def test_get_a_user(self):
+        user_1 =  User(fb_id='123', email=None, name='Jake')
+        db.session.add(user_1)
+        db.session.commit()
+
+        resp, status_code = user_service.get_a_user(1)
+
+        assert status_code == 201
+
+    @cleared_db
+    @patch('app.main.service.user_service.get_current_user')
+    def test_get_user_subscribers(self, mock):
+        user_1 =  User(fb_id='123', email=None, name='Jake')
+        user_2 = User(fb_id='123', email=None, name='Jane')
+        
+        user_1.followers.append(user_2)
+        db.session.add(user_1)
+        db.session.add(user_2)
+        db.session.commit()
+
+        mock.return_value = user_2
+        user_service.confirm_user_follow_request({"id": 1})
+
+        sub_list, status_code = user_service.get_user_subscribers()
+        assert status_code == 201
+    @cleared_db
+    @patch('app.main.service.user_service.get_current_user')
+    def test_get_user_subscribers(self, mock):
+        user_1 =  User(fb_id='123', email=None, name='Jake')
+        user_2 = User(fb_id='123', email=None, name='Jane')
+        
+        user_1.followers.append(user_2)
+        db.session.add(user_1)
+        db.session.add(user_2)
+        db.session.commit()
+
+        mock.return_value = user_2
+        user_service.confirm_user_follow_request({"id": 1})
+
+        sub_list, status_code = user_service.get_user_subscribers()
+        assert status_code == 201
```

Comment:
Over here our "current_user" needs to be user_1 so that get_user_subscribers returns a list that has user_2.
Response object for this request is:
[
  {
    "name": "Jane",
    "follower_id": 1
  }
]

Basically I think we should be asserting that the list is what we expect, otherwise its good